frontend/src/search-result.css

```diff
@@ -0,0 +1,7 @@
+.search-result-dialog {
```

Comment:
[Open Discussion, Not todo in this pull request]
There're a lot of css files in the src/ dir. Shouldn't this be migrated to another directory (e.g. `src/css/`) for sanitizing `src/` ?